lib/v2/njust/utils.js

```diff
@@ -0,0 +1,35 @@
+const delay = (timeout = 1000) => new Promise((resolve) => setTimeout(resolve, timeout));
+
+async function getContent(url, pptr = false) {
+    if (pptr) {
+        const browser = await require('@/utils/puppeteer-extra')();
```

Comment:
```suggestion
        const browser = await require('@/utils/puppeteer')({ stealth: true });
```
Added puppeteer with optional extra options(stealth only for now) in https://github.com/DIYgod/RSSHub/pull/9676. They shouldn't be separated. 
Also, tests are needed when adding new utility.

Reply:
done